tests/testBackground.py

```diff
@@ -218,7 +183,8 @@ def testRamp(self):
         bctrl.getStatisticsControl().setNumIter(1)
         backobj = afwMath.cast_BackgroundMI(afwMath.makeBackground(rampimg, bctrl))
 
-        print(rampimg.getArray())
+        if display:
+            print(rampimg.getArray())
```

Comment:
We can delete this: print statements in tests should not be run by default, which is why I wrapped it in the "debugging" variable here.